src/cli/parse-server.js

```diff
@@ -40,6 +41,10 @@ runner({
       process.exit(1);
     }
 
+    if (options.defaultResultOrder && dns.setDefaultResultOrder) {
```

Comment:
Is it even possible that `dns.setDefaultResultOrder` is `false`? It's supported from Node 14.8 and Parse Server 6 supports Node >= 14.21.0